packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -93,6 +93,13 @@ export function PreferencesModal( { isModalActive, isViewable, closeModal } ) {
 					help={ __( 'Make the editor look like your theme.' ) }
 					label={ __( 'Use theme styles' ) }
 				/>
+				<EnableFeature
+					featureName="showBlockBreadcrumbs"
+					help={ __(
+						'Shows block breadcrumbs at the bottom of the editor.'
+					) }
+					label={ __( 'Display block breadcrumbs' ) }
+				/>
```

Comment:
I do wonder if this should be conditionally shown by tieing it to the same underlying conditions of `shouldShowBlockBreadcrumbs` above.
```
                 ! hasReducedUI &&
		! isMobileViewport &&
		isRichEditingEnabled &&
		mode === 'visual';
```
The thought being that if `shouldShowBlockBreadcrumb` would be `false` regardless of the state of our `showBlockBreadcrumbs` setting, then having it visible and available to toggle may create confusion since it would never show up in those circumstances. 🤔   

Anyways, this seems like more of a concern that could be addressed in a follow up if deemed necessary.

Reply:
This makes sense to me! Pushing a commit to see it in action.